migrations/20230623168754_add_repo_to_release.sql

```diff
@@ -0,0 +1,2 @@
+alter table releases
+  add repository text default null;
```

Comment:
I would have thought a foreign key to the package_index_id would be the most appropriate

Reply:
Updated. This does mean we have to add the package index without a timestamp before we start importing releases else you get foreign key violations. We could take the approach of adding/updating the timestamp as we import each release but I think this is probably better.

Reply:
Indeed it's not stupid to have package indexes that we could know about before having importing them, especially if we want to provision the database with public stuff like hackage & chap (and others in the future).